src/libs/modals/modals/ModalDuplicateStrategy/ModalDuplicateStrategy.tsx

```diff
@@ -7,22 +7,50 @@ import { ModalFC } from 'libs/modals/modals.types';
 import { ModalOrMobileSheet } from 'libs/modals/ModalOrMobileSheet';
 import { Strategy } from 'libs/queries';
 import { getUndercutStrategy } from './utils';
+import { isOverlappingStrategy } from 'components/strategies/common/utils';
+import { useNavigate } from '@tanstack/react-router';
+import { getRoundedSpread } from 'components/strategies/overlapping/utils';
+import { StrategyType } from 'libs/routing';
 
 export type ModalDuplicateStrategyData = {
   strategy: Strategy;
 };
 
+const getStrategyType = (strategy: Strategy): StrategyType => {
+  const isBuyEmpty = !+strategy.order0.endRate;
+  const isSellEmpty = !+strategy.order1.endRate;
+  if (isBuyEmpty || isSellEmpty) return 'disposable';
```

Comment:
Could you use `isDisposableStrategy` from `src/components/strategies/common/utils.ts` here?